### ISSUE TITLE
Feature Request:  per capita rates

### ISSUE THREAD

**Issue by damonhaas** [aatishb/covidtrends#203]
These graphs are so helpful to show people where we are on our curve.  They can also be a big misleading for people to understand how the curve is doing by looking at total numbers rather than per capita for countries or states.  I think it would not be much work to add populations to have the option of showing the data per capita rather than just with totals. As an example I believe that right now Georgia has a worse per capita problem than California but because California is so much bigger it is difficult to illustrate that.

**Comment by dellena-bluefin** [comment]
Covid Trends is a great site!  Per capita would be so useful.

**Comment by scottmsul** [comment]
Yes please do this! I was making some plots of this on my own, see here: https://github.com/scottmsul/coronavirus

We can see that most countries end with around 10^-4  - 10^-3 total deaths, so it gives a reference for when herd immunity seems to kick in. But South Korea/New Zealand ended with less than 10^-5 so they probably stopped the spread before herd immunity.

Also the log-log plot shows that the US seems to be doing better per-capita wise than both Sweden and the UK, which you would never expect looking at just the total deaths.

This is very much needed in order to compare countries apples-to-apples.

**Comment by rpkoller** [comment]
@damonhaas the per capita feature was already requested in https://github.com/aatishb/covidtrends/issues/1 and https://github.com/aatishb/covidtrends/issues/30 and about the possible implementation and what speaks currently against it @aatishb wrote a reply in here: https://github.com/aatishb/covidtrends/issues/30#issuecomment-607927252

@scottmsul  i highly highly doubt that any country on earth is anywhere near herd immunity at all.  you argue that the countries in the middle right are already near herd immunity with deaths falling exponentially: 
![lower_right](https://user-images.githubusercontent.com/1665422/86769838-11657580-c050-11ea-8bd5-33e9e8c81dd1.png)
in that list you have for example spain... there is a recent study in the lancet contradicting your conclusion. there they get to a seropositivity of 5% tops and that is even for the hotspot areas: https://twitter.com/CT_Bergstrom/status/1280356720955420674
and you also have to keep the lead time bias in mind when interpreting the curves of new and total deaths: https://twitter.com/EpiEllie/status/1280305393516904448

**Comment by scottmsul** [comment]
Thanks for the response @rpkoller. I looked through [#1](https://github.com/aatishb/covidtrends/issues/1) and found someone has already implemented this and it's available [here](http://htmlpreview.github.io/?https://raw.githubusercontent.com/mmcguinn/covidtrends/wip/index.html). It looks like several US states have indeed passed 10^-3 deaths (Massachusetts, Connecticut, New Jersey, New York), but their trajectory has rounded out the top and falling which is promising. However this does make me re-think all those countries in Europe that I plotted earlier: if some US states passed through 10^-3, then maybe the places less than that don't have herd immunity. Or do they?

To be honest I'm more puzzled than anything, and not super convinced one way or the other. It feels like some of the pieces of data contradict others, and everything has caveats and biases. Yes I did see the study about only 5% testing positive for antibodies. There was also a study [here](https://www.cbc.ca/news/health/asymptomatic-covid-19-1.5629172) where 40% of asymptomatic cases had zero detectable antibodies after eight weeks. So it seems that people who did have the virus can lose their antibodies. Does this mean it's possible to gain, and then lose, herd immunity? Not necessarily, since T-cells can "remember" how to create the antibodies, even if the antibodies themselves are long-gone. But then again maybe the coronavirus will evolve into something that the T-cells don't know about, which is why flu shots are updated every two years. But maybe by then the coronavirus could also evolve into something less deadly, which is what happened with the Spanish Flu.

I'll admit that I could easily be wrong about what I said earlier about herd immunity, but I am by no means "highly highly" sure of anything at this point. I think the real test of herd immunity would be if one of the regions that followed the pattern of "rounding the top" and has fallen significantly and has deaths around ~10^-3 gets another major outbreak in several months that's as bad as its first wave, which based on the data from this app hasn't happened yet. But I'll be sure to follow these plots closely.

Apologies if this comes across as argumentative, I'm just trying to make sense of the data.

**Comment by rpkoller** [comment]
well all those regions that "rounded their top" of the curve, that process hasn't taken place in a natural way due to a high enough number of immune people aka no new hosts for the virus, but due to measurements like social distancing, masks and so forth. if you would let the virus go on without any measures there wouldn't been any rounding of the top yet in any of those regions. and you see that more or less in each and every country which loosens the measures of masks and distancing reopening their country too early is observing that their curves go back to linear or even exponential growth. 
e.g. 
https://aatishb.com/covidtrends/?region=US&location=Florida
https://aatishb.com/covidtrends/?location=Israel
https://aatishb.com/covidtrends/?location=Austria

and if the per capita in your linked example is cases per 1 millionen persons then for the example of new york either the dark figure of infected and asymptomatic people is astonishingly high or new york isnt anyway near to herd immunity anytime soon. i've calculated for absolute number with a population of 8,399 million and the latest infection count of 402388 you get a percentage of 4,79% ( (402388*100)/8399000 ) while for per capita from your link you get even less with a percentage of 2,06%. ( (20681,97*100)/1000000 ) - no idea why those percentages differ)...

**Comment by WaltJohnsonIII** [comment]
Per capita would be so useful...No..It is actually CRITICAL.  As stated below, the largest State or the largest Countries OF COURSE are going to have the most cases and deaths - that does not tell you the intensity, the power of the virus and how far it has gotten into the population.   PLEASE ADD PER CAPITA DATA.
Very clever, great data analytics!!

**Comment by scottmsul** [comment]
@bluezango - The author has already stated he doesn't want to add it, his reasoning is [here](https://github.com/aatishb/covidtrends/issues/30#issuecomment-607927252). Although someone did make a working per-capita version. It was created by user [daald](https://github.com/daald) who posted in issue [#1](https://github.com/aatishb/covidtrends/issues/1), link to the live version is [here](https://aatishb.com/covidtrends/?data=deaths&region=US&location=New+York). I've been personally using it to track case and death counts per capita, and it works great.

By the way... @rpkoller what are your thoughts on Sweden? They never locked down and their deaths are approaching zero. How could this not be herd immunity?
![image](https://user-images.githubusercontent.com/5205428/89246516-8765e980-d5c8-11ea-914a-a987a51eac25.png)

**Comment by rpkoller** [comment]
@scottmsul hmmm if you take a look at worldmeters for example >75k are still listed as active cases  with 37 critical and none is listed is actually recovered except the 5744 dead. the only thing you can say is that the confirmed cases on covidtrends look like they have a shrinking up to rather contained growth rate for the moment and the number of deaths "stagnates" relatively on the death curve... but the deaths arent approaching zero... in covidtrends it lists 44 deaths within the last 7 days... to deviate a "herd immunity" just based on the numbers from a distance i highly doubt ( how cases and deaths are counted, how is tested, how people behave, how is tourism, population density compared to distribution of cases over the country  etc pp - btw the worldmeter numbers and a population number of 10,23 million taken from google you have 0,791906158% infected, herd immunity starts at ) . and still the weekly confirmed case count in sweden is at 1617 on covid trends....

**Comment by aatishb** [comment]
Hi all, 

Apologies that I have been out of the loop for so long. Have had a bunch of work and personal stuff come up in the intervening months and haven't been able to prioritize maintaining Covid Trends. I'm grateful that so many folks have continued to find this resource useful, and for all your thoughtful input on here.

I just wanted to say that although I initially had a fairly strong stance against adopting per capita numbers, I can understand why this would be helpful, especially when comparing US states, or European countries. Plus, it is by far the most common request that I receive, and now that all countries are well into their growth curves, I agree that per capita is more reasonable that it would have been at an early growth stage.

I think we can do a per capita mode as a toggle switch on the menu, disabled by default. That way people can decide what view they prefer. If someone would like to put in a PR to this effect (or perhaps we can build off an existing PR), we can work on incorporating it.

@rpkoller let me know how this sounds to you.

**Comment by rpkoller** [comment]
@aatishb no worries, glad you made it through the work and personal stuff! about the per capita i am ok with adding it while i am still doubtful about the benefit ;))) but as you said still the most requested feature addition. and yes i agree i would strongly opt for disabled by default as you suggested! about the implementation. there is [one demo](http://raw.githack.com/jwosty/covidtrends/per-capita/index.html) utilizing a select box (absolut & per capita) while [another one](http://htmlpreview.github.io/?https://raw.githubusercontent.com/mmcguinn/covidtrends/wip/index.html) uses a checkbox... I would choose the selectbox, checkboxes are used so far for boolean show or hide something...
